lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/ashrae_90_1_prm.UserData.rb

```diff
@@ -145,6 +149,7 @@ class UserDataCSVBuilding < UserDataCSV
   def initialize(model, save_dir)
     super
     @component_name = 'Building'
+    @unique_model_object = true
```

Comment:
Building is a Unique Model Object. so getBuilding returns a single object not an array, and @components.each in write_csv will logically fail.